store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10560,4 +10560,9 @@ TODO: delete them permanently from here
   <defaultCOSValue>FALSE</defaultCOSValue>
   <desc>Feature to enable delivery status notification</desc>
 </attr>
+
+<attr id="4136" name="zimbraFeaturePasswordExpiryReminderEnabled" type="boolean" cardinality="single" optionalIn="cos,account" flags="accountInherited,accountInfo" since="10.1.4">
```

Comment:
don't think we can release a new ldap attribute in patch versions

Reply:
This is being checked to get released.

Reply:
@rcyarrapothu Please follow this ticket https://synacor.atlassian.net/browse/ZCS-16186 . Its till under discussion and validation.